Environmental/new_crawlers/water_quality_cork/elastic.py

```diff
@@ -0,0 +1,27 @@
+from elasticsearch import Elasticsearch
+
+
+class ElasticSearchClient:
+    def __init__(self, host: str, port: str, index: str):
+        self.es = Elasticsearch([{'host': host, 'port': port}])
+        self.index = index
+
+    def insert_doc(self, doc_: dict, id_: str = None) -> str:
+        res = self.es.index(index=self.index, body=doc_, id=id_)
+        return res['result']
+
+    def create_geomapping(self):
```

Comment:
this seems to simply create an index given a mapping.

what if i wanted to have a mapping that includes a geopoint and a double? or anything else that requires tampering.

how would we handle this?

i believe that in order for this to be useful, it would have to be more complex than simply adding a predefined mapping.

I am thinking that maybe each consumer should declare its own mapping because it knows what data to insert to the index